src/App.scss

```diff
@@ -575,6 +575,14 @@
   width: 16px;
 }
 
+.uni-FiltersButton-Count {
+  background: rgba(255, 255, 255, 0.32);
+  border-radius: 10px;
+  margin-left: 0.5rem;
+  min-width: 24px;
```

Comment:
Is it necessary?

Reply:
Yep because with short-width numbers (e.g., `1`) it looks weird when corders are too rounded.